modules/zio/logging/src/test/scala/tofu/logging/zlogs/ZLogsSuite.scala

```diff
@@ -37,6 +39,34 @@ class ZLogsSuite extends AnyFlatSpec with Matchers {
       LogTree(ctx)
     } should ===(List.fill(2)(expected))
   }
+
+  "ZLogs plain with context" should "log the context" in {
+    type CtxService = FiberRef[(FooService, BarService)]
```

Comment:
Is it possible to rework this spec? It's kinda confusing when I see context which is named as some service or something.

Reply:
> context which is named as some service
Look, actually `FiberRef[(FooService, BarService)]` is a service providing the context `(FooService, BarService)`, right?
From ZIO docs: 
> Has[A] represents a dependency on a service of type A
And `val logLayer: URLayer[Has[CtxService], Has[ZLogging.Make]]` below depends on this service
In production the service would be more complex like
```
trait CtxService {
  def getContext: UIO[(GlobalSate, FiberState)]

  def setReadiness(v: Boolean): UIO[Unit]

 //... etc
}
```